src/containers/AssetList.js

```diff
@@ -65,52 +65,50 @@ const assetData = [
 ];
 
 const TITLES = {
-  '/assets/dept': 'Assets: My department',
-  '/assets/all': 'Assets: All',
+	'/assets/dept': 'Assets: My department',
+	'/assets/all': 'Assets: All',
```

Comment:
I think the consistent indent size should actually be 2 to match, e.g. Sidebar.js, Static.js, AssetFormHEader.js. Maybe add a [*.js] section to .editorconfig to this effect?

Reply:
Spaces or tabs?

Reply:
Ignore that last comment, changes my tab size to 2 spaces.

Reply:
Nooo..... spaces! If you use a tab size of 2, GitHub will still display them as 8.